test/functional/notes_controller_test.rb

```diff
@@ -49,6 +49,67 @@ class NotesControllerTest < ActionController::TestCase
       end
     end
 
+    context "when an Important note is provided" do
```

Comment:
I don't think we need to capitalise the word here (it isn't anywhere else).
```suggestion
    context "when an important note is provided" do
```